src/compiler/tsconfig.json

```diff
@@ -1,13 +1,5 @@
 {
-  "compilerOptions": {
-    "allowJs": true,
-    "checkJs": true,
-    "noEmit": true,
-    "paths": {
-      "#compiler": ["./"],
-      "#compiler/*": ["./*"]
-    }
-  },
-  "include": ["**/*.js", "**/*.ts"],
-  "exclude": ["node_modules"]
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {"baseUrl": "../.."},
+  "include": ["**/*.js", "**/*.d.ts", "../core/**/globals.d.ts"],
```

Comment:
`tsconfig` can/should look identical to this in every subfolder of `src`, which is very satisfying